core/src/main/scala/chisel3/experimental/Probe.scala

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3.experimental
```

Comment:
Meta-concern: I am fine with dropping this `experimental` bit. This is intended to be super-mainline Chisel and we're only going to create churn for ourselves by putting this in `experimental` and then moving it later. What will likely happen is that this is put in `experimental` and then never moves. I'm in favor of this being in `chisel3._`.